### PR TITLE
[tools/circle_plus_gen] Add tparam inject feature

### PR DESCRIPTION
**Pull request by zetwhite**
This PR adds a feature
 - inject training parameters to a circle plus instance.
 - export circle plus instance as a *.circle file

ONE-DCO-1.0-Signed-off-by: seunghui youn <sseung.youn@samsung.com>

* related issue : https://github.com/Samsung/ONE/issues/12650

### CODE REVIEW

**Comment by zetwhite** [comment]
This will be opened after https://github.com/Samsung/ONE/pull/13012 merged

**Review by hseok-oh — DISMISSED** [review]
LGTM

File: tools/circle_plus_gen/lib/circle_plus.py [hunk 1]
```diff
@@ -7,6 +8,7 @@
 class CirclePlus():
     ''' Wrapper class of circle_schema_generated.ModelT'''
     TINFO_META_TAG = "CIRCLE_TRAINING"
+    CIRCLE_IDENTIFIER = b"CTR0"
```
Comment (jyoungyun) [inline tools/circle_plus_gen/lib/circle_plus.py:11]:
```suggestion
    CIRCLE_IDENTIFIER = b"CIR0"
```

File: tools/circle_plus_gen/lib/train_param.py [hunk 2]
```diff
@@ -7,6 +8,7 @@
 
 class TrainParam():
     '''Wrapper class of circle_traninfo_generated.ModelTrainingT'''
+    TRAINING_PARAM_IDENTIFIER = b"CTR0"
```
Comment (zetwhite) [inline tools/circle_plus_gen/lib/train_param.py:11]: (note) 

https://github.com/Samsung/ONE/blob/14333cd31f65c6447ee961f519ff7f9368214f13/runtime/libs/circle-schema/include/circle_traininfo.fbs#L18-L21

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**